mono/tests/Makefile.am

```diff
@@ -1627,6 +1629,15 @@ testbundle: console.exe
 	@$(with_mono_path) MONO_CFG_DIR=$(mono_cfg_dir) ./a.out >> mkbundle.stdout
 	@- rm -rf a.out
 
+test-verbose: verbose.exe
+	@echo "Testing MONO_VERBOSE_METHOD"
+	$(Q) MONO_VERBOSE_METHOD='*:Method' $(RUNTIME) --compile-all verbose.exe
```

Comment:
How do you verify that the output is correct? I would suggest having a `verbose.exe` test which will launch a subprocess with the appropriate `MONO_VERBOSE_METHOD` env variable defined and which verifies the output of the subprocess. You can then run this `verbose.exe` as part of the `test-jit` command, allowing it to integrate with NUnit.

Reply:
Sure, I'll take a look over the weekend.